package.json

```diff
@@ -1,149 +1,65 @@
-  "license": "EPL-2.0",
-  "repository": "https://github.com/zowe/zowe-cli-scs-plugin/",
-  "keywords": [
-    "credential manager",
-    "secure store",
-    "password"
-  ],
-  "publishConfig": {
-    "registry": "https://zowe.jfrog.io/zowe/api/npm/npm-local-release/"
-  },
-  "files": [
-    "lib",
-    "scripts"
-  ],
-  "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
-  "imperative": {
-    "configurationModule": "lib/imperative/configuration.js"
-  },
+  "private": true,
   "scripts": {
-    "clean": "rimraf lib",
-    "checkTestsCompile": "echo \"Checking that test source compiles...\" && tsc --project __tests__/test-tsconfig.json --noEmit ",
-    "circularDependencyCheck": "madge -c lib",
-    "lint": "eslint \"src/**/*.ts\" \"**/__tests__/**/*.ts\"",
-    "lint:src": "eslint \"src/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
-    "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
-    "prebuild": "npm run clean && npm run lint && echo Using TypeScript && tsc --version",
-    "build": "tsc --pretty && npm run checkTestsCompile && npm run circularDependencyCheck",
-    "prepublishOnly": "npm run build && bash ./scripts/bundleKeytar.sh",
-    "test": "npm run test:unit",
-    "test:unit": "env-cmd __tests__/__resources__/env/unit.env jest --coverage --testPathIgnorePatterns \".*/__system__/.*\" ",
-    "test:system": "env-cmd __tests__/__resources__/env/system.env jest .*/__system__/.* --coverage false --runInBand",
-    "coverage": "jest --coverage",
-    "watch": "tsc --pretty --watch",
-    "watch:test": "jest --watch",
-    "installPlugin": "npm install && npm run clean && npm run build && zowe plugins install .",
-    "postinstall": "node scripts/postinstall.js && echo \"The kubernetes secrets plugin has been successfully installed. The CLI will now use kubernetes secrets for storing credentials on your logged in cluster. CLI Profiles that you have created will need to be updated because their credentials are stored in plain text and not accessible from the credential manager.\" 1>&2",
-    "preuninstall": "node scripts/preuninstall.js",
-    "postuninstall": "echo \"CLI Profiles that you have created will need to be recreated because their credentials are stored in the credential manager that has been uninstalled. Refer to the available options in the help text to create a new profile 'zowe profiles create zosmf-profile -h'.\" 1>&2",
-    "typedoc": "typedoc --out ./docs/typedoc/ ./src/ --disableOutputCheck",
+    "build": "npm run build --workspaces",
+    "test": "npm run test --workspaces",
+    "test:system": "npm run test:system --workspaces",
+    "test:unit": "npm run test:unit --workspaces",
+    "lint": "npm run lint --workspaces",
+    "lintErrors": "npm run lintErrors --workspaces",
     "prepare": "husky install"
   },
-  "dependencies": {
-    "@kubernetes/client-node": "0.18.1"
+  "workspaces": [
+    "./packages/*"
+  ],
+  "engines": {
+    "npm": ">=9.4.2",
+    "node": ">=16.19.0"
```

Comment:
I know this is the top-level package so it should only affect developers, but do we want to be a bit more lenient on the versions required?
* Node 14.x or newer since Zowe supports all active LTS releases
* npm 7.x or newer since that's when workspace support was added